src/Microsoft.AspNetCore.SignalR.Core/DynamicHub.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// A base class for a dynamic SignalR hub.
```

Comment:
~~dynamic~~ SignalR hub that uses [dynamic](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/types/using-type-dynamic) to represent client invocations

Could we link to that docs page on dynamic? I don't remember if XML docs support that kind of link...

Reply:
I'll `<c>dynamic</c>`